src/app/dataset-block/metadata-block/components/event-details/builder.events.ts

```diff
@@ -170,9 +182,105 @@ class SetPollingSourceSectionBuilder extends EventSectionBuilder<SetPollingSourc
     }
 }
 
+class SetTransformSectionBuilder extends EventSectionBuilder<SetTransform> {
+    private readonly UNSUPPORTED_ROW_DESCRIPTOR: EventRowDescriptor = {
```

Comment:
This property looks very similar to the one in other event, think how to unify this. Maybe a property in the base class that invokes abstract method, or maybe a generic string.

Reply:
Moved to abstract class